demo/src/app/components/datepicker/demos/popup/datepicker-popup.html

```diff
@@ -0,0 +1,14 @@
+<form class="form-inline">
+  <div class="form-group">
+    <div class="input-group">
+      <input class="form-control" placeholder="yyyy-mm-dd"
+             name="dp" [(ngModel)]="model" ngbDatepicker #d="ngbDatepicker">
+      <div class="input-group-addon" (click)="d.toggle()" >
+        <img src="img/calendar-icon.svg" style="width: 1.2rem; height: 1.2rem; cursor: pointer;"/>
```

Comment:
Here, on Firefox and Chrome, the height of the addon is a bit too large. Setting the height to 1 rem makes it perfectly aligned with the input.